docs/eventing/samples/kafka/README.md

```diff
@@ -0,0 +1,28 @@
+The following examples will help you understand how to use the different Apache Kafka
+components for Knative to describe
+
+## Prerequisites
+
+All examples require:
+
+- A Kubernetes cluster with
+  - Knative Eventing v0.8+
+  - Knative Serving v0.8+
+
+- Apache Kafka cluster
+
+For Kubernetes, a simple Apache Kafka installation can be done by using Strimzi. Check out the [Quickstart](https://strimzi.io/quickstarts/) guides for both Minikube and Openshift. You can also install Kafka on the host.
```

Comment:
I want to remove 'can' in "...can be done by using Strimzi" since a new user might read that to mean its optional. To help clarify what's optional and whats required we should use a list (bulleted) to define each possible path. I also noticed that Apache Kafka refers to "a cluster", which conflicts with "Kubernetes clusters" so ive tried to make that clearer too.

[lines 14 - 20] From the Strimzi guide and Apache Kafka site, it sounds like there are separate "options" for each example, so maybe we can change to the following (written with the assumption that you will soon have more that just two Kafka examples):

---

You must have an [Apache Kafka cluster](https://kafka.apache.org/intro) running. Depending on the example
that you want to run, you have the following options:

- For all examples: You can use [Strimzi](https://strimzi.io/quickstarts/), an Apache Kafka Operator for Kubernetes, Minikube, and Openshift.

- For "*`KafkaSource` to Knative Serving Service*": You have the option to use a [Apache Kafka server](https://kafka.apache.org/quickstart).

Reply:
LGTM

Reply:
@RichieEscarez 

> For "KafkaSource to Knative Serving Service": You have the option to use a Apache Kafka server.

No, the cluster needs to be accessible.... You need one installation of Kafka, and simplest option is to install one into your k8s (minikube) cluster.  Reaching outside is complexer....

Reply:
This is the statement that is confusing me:

> Note: The `KafkaSource` is not limited to Apache Kafka installations on
> Kubernetes. It is also possible to use an off-cluster Apache Kafka
> installation.

With this statement in the prerequisites section, it sounds like an option. But if this is not covered in this guide, it should be removed.